### ISSUE TITLE
Lors de l'ajout d'un node, il se crée toujours à la fin de la liste

### ISSUE THREAD

**Issue by mstp** [etienne/dialog-edit#20]

**Comment by etienne** [comment]
Oui, pour l'instant ils s'affichent dans l'ordre de création. Je vais éventuellement permettre de les réordonner. Comment tu aurais vu ça? Genre que le nouveau node s'affiche après celui qui est présentement sélectionné?

**Comment by mstp** [comment]
Oui, intuitivement je pensais que le nouveau node se créerait juste en-dessous du node sélectionné.
